src/main/java/com/example/bookstore/mapper/ShoppingCartMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.example.bookstore.mapper;
+
+import com.example.bookstore.config.MapperConfig;
+import com.example.bookstore.dto.shoppingcart.ShoppingCartDto;
+import com.example.bookstore.model.ShoppingCart;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(config = MapperConfig.class, uses = CartItemMapper.class)
+public interface ShoppingCartMapper {
+    @Mapping(source = "user.id", target = "userId")
+    @Mapping(source = "cartItems", target = "cartItems")
+    ShoppingCartDto toDto(ShoppingCart shoppingCart);
+
+    @Mapping(source = "userId", target = "user.id")
+    @Mapping(source = "cartItems", target = "cartItems")
```

Comment:
try to keep the same style, in CartItemMapper you used Mappings wrapper annotation